addons/mentions/userstyle.css

```diff
@@ -0,0 +1,12 @@
+.sa-mention {
+  background-color: var(--darkWww-button, #4d97ff);
+  color: var(--darkWww-button-text, white) !important;
```

Comment:
```suggestion
  background-color: var(--mentions-color);
  color: var(--mentions-button-text) !important;
```

Reply:
Thank you for letting me know about the issue with disallowing edits, I've allowed them now - I just didn't want to risk anything.